run_tests.sh

```diff
@@ -23,7 +23,7 @@ fi
 
 set -x
 
-ADDR=${HOST:="http://127.0.0.1:8006"}
+HOST=${HOST:="http://127.0.0.1:8006"}
```

Comment:
Let's use `http://$ADDR:$PORT` since we have two separate environment variables.